### PR TITLE
Add NuGet repo support matrix

### PR DESCRIPTION
**Pull request by sdwheeler**
Add NuGet repo support matrix

- Fixes [AB#116629](https://dev.azure.com/msft-skilling/cebd7ef5-4282-448b-9701-88c8637581b7/_workitems/edit/116629)

### CODE REVIEW

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
+---
+description: This article lists the repositories that have been tested with PowerShellGet v3 and how to configure them.
+ms.date: 06/16/2023
+title: Supported repository configurations
+---
+# Supported repository configurations
+
+The **Microsoft.PowerShell.PSResourceGet** module, also known as **PowerShellGet v3**, works with
+NuGet package repositories andlocal file stores. In general, the cmdlets should work with any NuGet
```
Comment (alerickson) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:9]:
```suggestion
NuGet package repositories and local file stores. In general, the cmdlets should work with any NuGet
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+repository that supports the NuGet protocol. However, not all NuGet repositories support all the
+features of PowerShellGet v3.
```
Comment (alerickson) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:11]:
```suggestion
features of PSResourceGet.
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+
+The following repositories have been tested with PowerShellGet v3.
```
Comment (alerickson) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:13]:
```suggestion
The following repositories have been tested with PSResourceGet.
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+## The PowerShell Gallery
+
+The PowerShell Gallery is the default repository for PowerShellGet v3. The Gallery is a NuGet
```
Comment (alerickson) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:17]:
```suggestion
The PowerShell Gallery is the default repository for PSResourceGet. The Gallery is a NuGet
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+repository that uses the NuGet v2 protocol.
+
+You don't need credentials to search, download, or install packages from the PowerShell Gallery. You
+must create an account and API key to publish packages to the PowerShell Gallery. For more
+information, see [Creating and publishing an item][01].
+
+The PowerShell Gallery supports all the features of PowerShellGet v3.
```
Comment (alerickson) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:24]:
```suggestion
The PowerShell Gallery supports all the features of PSResourceGet.
```

**Review by alerickson — APPROVED** [review]
Looks good, mostly just suggested changes for changing 'PowerShellGet v3' to 'PSResourceGet'

**Review by SydneyhSmith — APPROVED** [review]
Looks great, thanks for putting this together

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 6]
```diff
@@ -0,0 +1,283 @@
+---
+description: This article lists the repositories that have been tested with PowerShellGet v3 and how to configure them.
+ms.date: 06/28/2023
+title: Supported repository configurations
+---
+# Supported repository configurations
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:6]:
```suggestion
---

# Supported repository configurations
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+
+The **Microsoft.PowerShell.PSResourceGet** module, also known as **PowerShellGet v3**, works with
+NuGet package repositories and local file stores. In general, the cmdlets should work with any NuGet
+repository that supports the NuGet protocol. However, not all NuGet repositories support all the
+features of PSResourceGet.
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:11]: I'm not sure about this phrase construction:

> ... any NuGet repository that supports the NuGet protocol

Maybe instead:

```suggestion
The **Microsoft.PowerShell.PSResourceGet** module, also known as **PowerShellGet v3**, works with
NuGet package repositories and local file stores. In general, the cmdlets should work with any
artifact repository that supports the NuGet protocol. However, not all NuGet repositories support
all the features of PSResourceGet.
```

or

```suggestion
The **Microsoft.PowerShell.PSResourceGet** module, also known as **PowerShellGet v3**, works with
NuGet package repositories and local file stores. In general, the cmdlets should work with any
repository that supports the NuGet protocol. However, not all NuGet repositories support all the
features of PSResourceGet.
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+You don't need credentials to search, download, or install packages from the PowerShell Gallery. You
+must create an account and API key to publish packages to the PowerShell Gallery. For more
+information, see [Creating and publishing an item][01].
+
+The PowerShell Gallery supports all the features of PSResourceGet.
+
+## NuGet.org
+
+NuGet.org is a public host of NuGet packages used by millions of .NET and .NET Core developers every
+day. The NuGet.org provides Gallery a NuGet repository that uses the NuGet v3 protocol.
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:29]: I'm not sure about the last sentence. Does NuGet.org provide a gallery, which is a NuGet repository, or something else?

```suggestion
NuGet.org is a public host of NuGet packages used by millions of .NET and .NET Core developers every
day. NuGet.org provides a NuGet repository that uses the NuGet v3 protocol.
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 9]
```diff
@@ -0,0 +1,283 @@
...
+  - `Find-PSResource -Name 'Az*'`
+  - `Find-PSResource -Name 'Az*' -Tag compute`
+- Search by command or DSC resource name, for example:
+  - `Find-PSResource -Command commandname`
+  - `Find-PSResource -DSCResource dscresourcename`
+
+## Azure Artifacts
+
+Azure Artifacts is a feature of Azure DevOps. Azure Artifacts enables developers to share their code
+efficiently and manage all their packages from one place. With Azure Artifacts, developers can
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:61]: Does it allow developers to manage their packages _from_ one place or _in_ one place? Both?

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 10]
```diff
@@ -0,0 +1,283 @@
...
+    Name = 'MyAdoFeed'
+    Uri  = 'https://pkgs.dev.azure.com/MyOrg/MyProject/_packaging/MyFeed/nuget/v3/index.json'
+}
+Register-PSResourceRepository @params
+```
+
+Azure DevOps allows you to create public or private feeds for your Azure Artifacts. You don't need
+credentials to search, download, or install packages from an Azure Artifacts public feed. To publish
+artifacts or access private feeds, you must have an account and an API key. For more information
+about, see [Connect to feed as a PowerShell repository][03].
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:83]: Either:
```suggestion
artifacts or access private feeds, you must have an account and an API key. For more information,
see [Connect to feed as a PowerShell repository][03].
```
Or:
```suggestion
artifacts or access private feeds, you must have an account and an API key. For more information
about getting an API key, see [Connect to feed as a PowerShell repository][03].
```

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 11]
```diff
@@ -0,0 +1,283 @@
...
+## GitHub Packages
+
+GitHub Packages is a software package hosting service that allows you to host your software packages
+privately or publicly and use packages as dependencies in your projects. For more information, see
+[Introduction to GitHub Packages][06].
+
+The GitHub Packages feed is a NuGet repository that uses the NuGet v3 protocol. The feed URI has the
+following format: `https://nuget.pkg.github.com/<namespace>/index.json`. Replace `<namespace>` with
+the name of the personal account or organization to which your packages are scoped. For example, use
+the following command to register an Azure Artifacts feed as a PSResource repository:
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:111]:
```suggestion
the following command to register an GitHub Packages feed as a PSResource repository:
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md (hunk 9 above)
Comment (sdwheeler) [reply powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:61]: stet

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**